runtime/gc_base/ObjectAccessBarrier.hpp

```diff
@@ -541,6 +541,17 @@ class MM_ObjectAccessBarrier : public MM_BaseVirtual
 		/* no-op default implementation */
 	}
 
+	virtual J9Object *referenceGet(J9VMThread *vmThread, J9Object *refObject)
+	{
+		return J9VMJAVALANGREFREFERENCE_REFERENT_VM(vmThread->javaVM, refObject);
+	}
+
+	virtual void referenceReprocess(J9VMThread *vmThread, J9Object *refObject)
+	{
+		/* Equivalent to J9WriteBarrierBatchStore */
+		preBatchObjectStore(vmThread, refObject);
+	}
```

Comment:
I think it's more correct to have these unimplemented in base, and implement them in Standard subclass.